server/Arcadia.Assistant.Calendar/CalendarEventsStorageBase.cs

```diff
@@ -77,23 +86,162 @@ protected override void OnCommand(object message)
                     }
                     break;
 
+                case ApproveCalendarEvent msg:
+                    try
```

Comment:
90 lines already. should we split it somehow? :)

Reply:
Probably, yes, but not now, ok? :)